linode/manager
# change: [M3-8377, M3-8579] - Move Region section above Images in Linode Create and update default OS to Ubuntu 24.04 LTS

## Description 📝
1. Move Region section above the Images section in the Linode Create flows. This change has been approved by UX/Marketing/Docs
2. Set Default OS to Ubuntu 24.04 LTS

No changes have been made in the Backups/Clone tab since they don't have an Images section

## Preview 📷
| Before  | After   |
| ------- | ------- | 
| ![image](https://github.com/user-attachments/assets/ea2c2d4a-3d1c-4705-8f0a-f4975aa93b4b) | ![image](https://github.com/user-attachments/assets/29135159-2d0c-484e-a446-5d0bb3d44f0d) |

## How to test 🧪
### Verification steps
(How to verify changes)
- Click through the Linode Create flow tabs and ensure that the Region section is above the Images section and that the default OS is Ubuntu 24.04 LTS
- There should be no regressions in creating a Linode

## As an Author I have considered 🤔

*Check all that apply*
